discovery/src/main/scala/akka/discovery/SimpleServiceDiscovery.scala

```diff
@@ -21,6 +21,7 @@ import scala.concurrent.duration.FiniteDuration
 @ApiMayChange
 object SimpleServiceDiscovery {
 
+  // TODO for use cases like aggregate it might be worth making this an ADT with a NoTargets case
```

Comment:
Maybe not worth it. Just some times it might make sense to distinguish between a service being there but no registered endpoints vs not being there at all e.g. in config, consul etc